mmv1/products/compute/api.yaml

```diff
@@ -1379,34 +1380,42 @@ objects:
+                           is selected in round robin order.
 
-          * LEAST_REQUEST - An O(1) algorithm which selects two random healthy
-                            hosts and picks the host which has fewer active requests.
+          * `LEAST_REQUEST`: An O(1) algorithm which selects two random healthy
+                             hosts and picks the host which has fewer active requests.
 
-          * RING_HASH - The ring/modulo hash load balancer implements consistent
-                        hashing to backends. The algorithm has the property that the
-                        addition/removal of a host from a set of N hosts only affects
-                        1/N of the requests.
+          * `RING_HASH`: The ring/modulo hash load balancer implements consistent
+                         hashing to backends. The algorithm has the property that the
+                         addition/removal of a host from a set of N hosts only affects
+                         1/N of the requests.
 
-          * RANDOM - The load balancer selects a random healthy host.
+          * `RANDOM`: The load balancer selects a random healthy host.
 
-          * ORIGINAL_DESTINATION - Backend host is selected based on the client
-                                   connection metadata, i.e., connections are opened
-                                   to the same address as the destination address of
-                                   the incoming connection before the connection
-                                   was redirected to the load balancer.
+          * `ORIGINAL_DESTINATION`: Backend host is selected based on the client
+                                    connection metadata, i.e., connections are opened
+                                    to the same address as the destination address of
+                                    the incoming connection before the connection
+                                    was redirected to the load balancer.
 
-          * MAGLEV - used as a drop in replacement for the ring hash load balancer.
-                     Maglev is not as stable as ring hash but has faster table lookup
-                     build times and host selection times. For more information about
-                     Maglev, refer to https://ai.google/research/pubs/pub44824
+          * `MAGLEV`: used as a drop in replacement for the ring hash load balancer.
+                      Maglev is not as stable as ring hash but has faster table lookup
+                      build times and host selection times. For more information about
+                      Maglev, refer to https://ai.google/research/pubs/pub44824
 
-          This field is applicable only when the load_balancing_scheme is set to
-          INTERNAL_SELF_MANAGED.
+
+          This field is applicable to either:
+
+          * A regional backend service with the serviceProtocol set to HTTP, HTTPS, or HTTP2, and loadBalancingScheme set to INTERNAL_MANAGED.
```

Comment:
would it be more helpful to have the field names be in snake_case?

Reply:
I wasn't sure what to do about that, since this is in an api.yaml file (so hypothetically agnostic to the terraform output... but maybe we don't care at this point?)

Reply:
Fair point. It is very likely just useful for TF, but not a big deal either way.

Reply:
It looks like the old text used the snake case version so I'll conform to that.